src/backend/mlir.rs

```diff
@@ -322,8 +322,8 @@ impl MlirBackend {
             ir::Control::Enable(ir::Enable { group, .. }) => {
                 write!(f, "calyx.enable @{}", group.borrow().name().id)
             }
-            ir::Control::StaticEnable(_) => {
-                panic!("StaticEnable not yet supported")
+            ir::Control::Static(_) => {
+                panic!("Static not yet supported")
```

Comment:
prefer `todo!` over `panic!` when it is something we would like to support in the future, otherwise use `unimplemented!` if it's something we don't plan to make functional in the proximate future. Save `panic!` for things that constitute a genuine runtime error in the system